JS Basics/intro.js

```diff
@@ -0,0 +1,20 @@
+let sentence = "I can eat bananas all day";
+let word = sentence.slice(10, 17);
+alert(word.toUpperCase());
+
+let cars = ["Saab", "Volvo", "BMW"];
+
+let bmw = cars[2];
```

Comment:
I would advise to use [`find`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) method if you need to find an element by some value